src/SectionParserData.h

```diff
@@ -51,6 +51,12 @@ namespace snowcrash {
         /** Table mapping named type to sub types */
         mson::NamedTypeInheritanceTable namedTypeInheritanceTable;
 
+        /** Table mapping named types to their dependent named types */
+        mson::NamedTypeDependencyTable namedTypeDependencyTable;
+
+        /** Variable to store the current named type */
+        mson::Literal namedTypeContext;
```

Comment:
I didn't like writing this here at all, but it was necessary. Maybe there is a better way.